packages/react-aria-components/docs/DropZone.mdx

```diff
@@ -0,0 +1,362 @@
+import styles from '@react-spectrum/docs/src/docs.css';
+import packageData from 'react-aria-components/package.json';
+import Anatomy from '@react-aria/datepicker/docs/daterangepicker-anatomy.svg';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {Divider} from '@react-spectrum/divider';
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Drag and drop
+keywords: [dropzone, drop, dnd, drag and drop, aria, accessibility]
+type: component
+---
+
+# Dropzone
+
+<PageDescription>{docs.exports.DropZone.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DropZone']} />
+
+## Example
+
+```tsx example
+import {DropZone} from 'react-aria-components';
+
+<DropZone>
+  Drop object here
+</DropZone>
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.react-aria-DropZone {
+  --border-color: var(--spectrum-alias-border-color);
+  --border-color-pressed: var(--spectrum-alias-border-color-focus);
+  --background-color: var(--spectrum-global-color-gray-50);
+  --background-color-pressed: var(--spectrum-global-color-gray-100);
+  --text-color: var(--spectrum-alias-text-color);
+  --focus-ring-color: var(--spectrum-alias-border-color-focus);
+
+  color: var(--text-color);
+  background: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  appearance: none;
+  vertical-align: middle;
+  font-size: 1.2rem;
+  text-align: center;
+  margin: 0;
+  outline: none;
+  padding: 24px 12px;
+  width: 25%;
+  display: inline-block;
+
+  &[data-drop-target] {
+    box-shadow: 0 0 0 1px var(--focus-ring-color);
+    background: var(--background-color-pressed);
+    border-color: var(--border-color-pressed);
+  }
+
+  &[data-focus-visible] {
+    border-color: var(--focus-ring-color);
+    box-shadow: 0 0 0 1px var(--focus-ring-color);
+  }
+}
+
+@media (forced-colors: active) {
+  .react-aria-DropZone {
+    forced-color-adjust: none;
+    --border-color: ButtonBorder;
+    --border-color-drop-target: ButtonBorder;
+    --background-color: ButtonFace;
+    --background-color-pressed: ButtonFace;
+    --text-color: ButtonText;
+    --focus-ring-color: Highlight;
+  }
+}
+```
+
+</details>
+
+## Features
+
+There is no native element to implement a dropzone in HTML that is widely supported. `DropZone` helps achieve accessible dropzone components that can be styled as needed. 
+
+* **Styleable** – Hover, keyboard focus, and the drop target state is provided for easy styling. These styles only apply when interacting with an appropriate input device, unlike CSS pseudo classes. 
+* **Accessible** – Uses a visually hidden `<button>` under the hood to allow interactions like dropping objects with a keyboard. 
+* **Flexible** – Can be used standalone or with `FileTrigger` to allow file uploads.
```

Comment:
I would call out the support for various data types too. FileTrigger is technically a separate component, and you can render whatever you want inside a drop zone.

```suggestion
* **Flexible** – Files, directories, and custom data types can be dropped, and the contents of the drop zone can be fully customized.
```